.github/workflows/ci.yml

```diff
@@ -5,12 +5,27 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
       - uses: ruby/setup-ruby@v1
```

Comment:
Slightly unrelated but it doesn't look like we're using Ruby in this workflow - or am I missing something?

Reply:
yeah, I guess it is to do with the Signon secret generation code, it runs some rspecs for it.

Reply:
Yeah I guessed the same, just couldn't find where :/